docpages/01_frequently_asked_questions.md

```diff
@@ -1,88 +1,113 @@
 
 ## Does this library support sharding?
+
 Yes! D++ supports sharding and also clustering (grouping of shards into one process) to ensure it is scalable for small and large bots alike.
 
 ## How do I contribute to the documentation and website?
+
 The documentation and website are built using Doxygen. To contribute to site pages, submit a Pull Request to the main repository. The site pages can be found within the `docpages` directory. Details of classes, variables, namespaces etc. are auto generated from Doxygen comments within the library source code in the `include` and `src` folders.
 
 ## What version of the Discord API does this library support?
+
 D++ only supports Discord API v10, the latest version. D++ major version numbers match the supported Discord API version.
 
 ## Does this Discord library support the threads feature?
+
 Yes! D++ supports Discord threads. You can create, edit and delete threads and also attach events watching for messages within threads.
 
 ## Does D++ require C++20 support?
+
 No, the library only requires C++17. We have some optional features such as \ref using-coroutines "coroutines" that do require C++20, but they are disabled by default.
 
 ## When I start my bot I get an error: "error while loading shared libraries: libdpp.so: cannot open shared object file: No such file or directory"
+
 To fix this issue, run `ldconfig`: `sudo ldconfig` as root. Log out if your SSH session and log back in, and the bot should be able to find the library.
 
 ## When compiling with voice support, I get an error: "No rule to make target 'sodium_LIBRARY_DEBUG-NOTFOUND', needed by 'libdpp.so'. Stop."
+
 The libsodium package requires pkg-config, but does not check for it when installed. Install it as root, e.g. `sudo apt install pkg-config`. Rerun cmake, and rebuild the library.
 
 ## When I try to instantiate a dpp::cluster in windows, a std::bad_alloc exception is thrown
-If this happens, ensure you are using the correct precompiled build of the library. Our precompiled binaries are built in two forms, **release mode** and **debug mode** for Visual Studio 2019/2022. These two versions of the library are not cross-compatible due to differences in the debug and release libstdc++. You should not need to build your own copy, but please see the section about \ref buildwindows for more information on how to build your own copy, if needed.
+
+If this happens, ensure you are using the correct precompiled build of the library. Our precompiled binaries are built in two forms, **release mode** and **debug mode** for Visual Studio 2019/2022. These two versions of the library are not cross-compatible due to differences in the debug and release STL (Microsoft standard library implementation). You should not need to build your own copy, but please see the section about \ref buildwindows for more information on how to build your own copy, if needed.
```

Comment:
this is scary myhead. master has fixed this so we can maybe rewrite this q to talk about the message box that will come up to directly tell the user they have a mismatch instead of some weird bad alloc or scary myhead

Reply:
Yeah, imo just the title needs renaming here